test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12118,6 +12143,8 @@ def cmp_fairshare_entities(self, name1=None, name2=None):
             name2 = str(name2)
 
         pbsfs = os.path.join(self.pbs_conf['PBS_EXEC'], 'sbin', 'pbsfs')
+        if self.sc_name is not 'default':
+            cmd += ['-I', self.sc_name]
```

Comment:
command is loaded to pbsfs here , So -I sched_name should be added to pbsfs , else move this if condition after next line